examples/mnist/run.py

```diff
@@ -14,8 +14,7 @@
 import os
 import pickle
 import time
-from urllib.error import URLError
-from urllib.request import urlretrieve
+import sys
```

Comment:
Please sort the imports alphabetically. This [python utility](https://github.com/timothycrosley/isort) might helps.

Reply:
Sure, I will add commit to this PR>

Reply:
Enable this option if you run isort from command line. 
```
-sl, --force-single-line-imports
                        Forces all from imports to appear on their own line
```
You can also create a `.cfg` file (see [here](https://github.com/timothycrosley/isort#configuring-isort)) with `force_single_line=True` in it.

Reply:
Do you really want that? You wrote the `Conv2D, Dense, Flatten, ReLU` imports in one line. This option does not change the code from my part.

Reply:
master (preferred)
```
from core.layers import Dense
from core.layers import Conv2D
from core.layers import Flatten
from core.layers import ReLU
```

your PR
```
from core.layers import Conv2D, Dense, Flatten, ReLU
```

Reply:
@borgwang Ah, I see. `isort` introduced that, no wonder I have no impression I have changed that piece. Sorry.